motoko/cert-var/src/cert_var_assets/src/index.js

```diff
@@ -0,0 +1,67 @@
+import { Actor, HttpAgent } from '@dfinity/agent';
+import { Certificate, makeNonceTransform } from '@dfinity/agent';
+import { cert_var } from '../../declarations'
+
+document.getElementById("certifyBtn").addEventListener("click", async () => {
+  const newVal = BigInt(document.getElementById("newValue").value);
+  const _ = await cert_var.set(newVal);
+  const resp = await cert_var.get();
+
+  // to do -- From where to get the port and host, in general?
+  const port = 8000;
+  const agent = await Promise.resolve(new HttpAgent({ host: 'http://127.0.0.1:' + port })).then(
+      async agent => {
+          await agent.fetchRootKey();
+          agent.addTransform(makeNonceTransform());
+          return agent;
+      },
+  );
+  const now = Date.now() / 1000;
+
+  const cert = new Certificate(resp, agent);
```

Comment:
@chenyan-dfinity 

When I try this out locally, I get this in the browser console:
```
decoder.js:570 Uncaught (in promise) Error: No valid result
    at Decoder._decode (decoder.js:570)
    at Decoder.decodeFirst (decoder.js:577)
    at Module.decode (cbor.js:94)
    at new Certificate (certificate.js:75)
    at HTMLButtonElement.<anonymous> (index.js:21)
 ```

Reply:
Certificate expects a blob, so it should be `resp[1][0]`, meaning the second return value and unwrapping the opt.

Reply:
No, nevermind.  When I use the dev tools, I see that it is not null.